lesson8/lesson8.js

```diff
@@ -33,12 +33,16 @@ function createList(item){
 
 function fetchListData() {
   showLoadingImg();
-  return new Promise((resolve) => {
-  setTimeout(() => resolve(listItems), 3000)
-  });
+  return new Promise(function(resolve, reject) {
```

Comment:
[NITS]
You can use the arrow function for this part as in the previous lesson.

Reply:
I was ashamed to not know the arrow function. Thank you very much for teaching me. I tried to fix it!
https://github.com/rikokko/to_be_frontend/pull/16/commits/2a5bb887997bb5ae20f910501ac8ac16526a9964